src/stats.c

```diff
@@ -556,10 +635,34 @@ stats_arena_lextents_print(emitter_t *emitter, unsigned i, uint64_t uptime) {
 		}
 
 		CTL_M2_GET("arenas.lextent.0.size", j, &lextent_size, size_t);
+		CTL_M2_M4_GET("stats.arenas.0.lextents.0.nmalloc", i, j,
+		    &nmalloc, uint64_t);
+		CTL_M2_M4_GET("stats.arenas.0.lextents.0.ndalloc", i, j,
+		    &ndalloc, uint64_t);
 		CTL_M2_M4_GET("stats.arenas.0.lextents.0.curlextents", i, j,
 		    &curlextents, size_t);
 
+		if (prof_stats_on) {
+			size_t prof_stats_len = sizeof(prof_stats_t);
+			prof_live_mib[3] = j;
+			xmallctlbymib(prof_live_mib, prof_stats_miblen,
+			    &prof_live, &prof_stats_len, NULL, 0);
+			prof_accum_mib[3] = j;
+			xmallctlbymib(prof_accum_mib, prof_stats_miblen,
+			    &prof_accum, &prof_stats_len, NULL, 0);
+		}
+
 		emitter_json_object_begin(emitter);
+		if (prof_stats_on) {
+			emitter_json_kv(emitter, "prof_live_r",
```

Comment:
Probably "prof_live_requested" and "prof_live_count" are better names for these keys?

Reply:
Yes. I was trying to make the column narrower, but, you're right, the names became too hard to understand. Expanded the name, as well as the column widths.